### PR TITLE
Home explore

### PR DESCRIPTION
**Pull request by JatuFaique**
- Added following user functionality
- filter home with respect to following only users.

### CODE REVIEW

File: src/Components/RightBar.js [hunk 1]
```diff
@@ -1,33 +1,70 @@
...
+import { followUser } from "../features/auth/authSlice";
 
 function RightBar() {
+  const { token, userDetail } = useSelector((state) => state.auth);
+  const [users, setUsers] = useState([]);
+  const dispatch = useDispatch();
+
+  const handleFollow = (userId) => {
+    const userInfo = { userId, token };
+    // console.log(userInfo);
```
Comment (Developers-codz) [inline src/Components/RightBar.js:13]: remove commented code

File: src/Components/RightBar.js [hunk 2]
```diff
@@ -1,33 +1,70 @@
...
+    dispatch(followUser(userInfo));
+  };
+
+  const getAllUsers = async () => {
+    try {
+      const response = await axios.get("api/users");
+      if (response.status === 200 || 201) {
+        // console.log("gg", response.data);
```
Comment (Developers-codz) [inline src/Components/RightBar.js:21]: here as well

File: src/features/auth/authSlice.js [hunk 3]
```diff
@@ -9,10 +9,55 @@ const initialState = {
...
+      const res = await axios.post(
+        `/api/users/follow/${userInfo.userId}`,
+        {},
+        {
+          headers: {
+            authorization: userInfo.token,
+          },
+        }
+      );
+      console.log(res.data);
```
Comment (Developers-codz) [inline src/features/auth/authSlice.js:26]: here as well

File: src/features/auth/authSlice.js [hunk 4]
```diff
@@ -85,6 +129,34 @@ export const authSlice = createSlice({
       state.status = "Rejected";
       state.login = false;
     },
+    [followUser.pending]: (state) => {
+      state.status = "loading";
+    },
+    [followUser.fulfilled]: (state, action) => {
+      console.log("ho gayi", action.payload);
```
Comment (Developers-codz) [inline src/features/auth/authSlice.js:136]: there are so many consoles do check and remove it if no more needed

**Review by Developers-codz — APPROVED** [review]
Good work on code and UI both 
Just added few suggestions, also  who to follow list should not contain currentuser check that, rest good to merge

**Review by naazweb — APPROVED** [review]
Please check and remove console logs and rest looks good to merge.

**Pull request merged**